src/App.test.js

```diff
@@ -0,0 +1,53 @@
+import calculate from './logic/calculate';
+
+describe('Test calculate', () => {
+  it('6 + 4 = 10', () => {
+    const obj = {
+      total: '6',
+      next: '4',
+      operation: '+',
+    };
+    const res = calculate(obj, '=');
+    expect(res.total).toBe('10');
+  });
+
+  it('6 - 4 = 2', () => {
+    const obj = {
+      total: '6',
+      next: '4',
+      operation: '-',
+    };
+    const res = calculate(obj, '=');
+    expect(res.total).toBe('2');
+  });
+
+  it('6 x 4 = 24', () => {
+    const obj = {
+      total: '6',
+      next: '4',
+      operation: 'x',
+    };
+    const res = calculate(obj, '=');
+    expect(res.total).toBe('24');
+  });
+
+  it('6 ÷ 4 = 1.5', () => {
+    const obj = {
+      total: '6',
+      next: '4',
+      operation: '÷',
+    };
+    const res = calculate(obj, '=');
+    expect(res.total).toBe('1.5');
+  });
+
+  it('6 % 4 = 2', () => {
+    const obj = {
+      total: '6',
+      next: '4',
```

Comment:
- [ ] Great for adding tests for calculating but put in your mind to add tests for operation also. And also as stated in the requirements you suppose to write unit tests for all React components by using jest snapshots and React Testing Library
>![image](https://user-images.githubusercontent.com/53402600/230378556-6403daf7-1a53-41ab-85ef-a738c897dcf6.png)